src/Button.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import styled from "styled-components";
+
+const StyledButton = styled.button`
+  text-align: center;
+  background: lightgrey;
+`;
+
+const Button = ({ children, ...props }) => {
```

Comment:
Spreading props is always a little difficult. It's perfectly fine to do so 
but the next developer will not know what props are being passed here. For readability 
it's probably better to name them explicitly

Reply:
Especially since in your case here you're not actually passing any props :)

Reply:
Agreed it's not needed here when no props are being passed at all, but I'd argue that for html elements with a long list of innate props it's good practice to spread them on.

Reply:
Developers in the future that add to your code may use your reusable Button component assuming that props are available that you haven't specifically declared.

Reply:
I'd actually argue that there is a use to passing props in here. While yes they should be naming the things that they are using, there could be other props that they don't necessarily know in advance and don't want to name. For example, disabled etc

Reply:
Also guys, I'd add some unit tests for your button :) For example, a snapshot of it by itself, a snapshot of it passing in children, maybe even a test to see if your handle click props is triggered when the button itself is clicked.

Reply:
Yes please, tests in every PR!

Reply:
And one last one 👀 Can we add the actual functionality of the button explicitly? Like pull the onClick prop out of the props and give it to the styledButton onClick?

Reply:
Snapshot tests are a good idea but testing if a component passes props is a little pointless. That's why we're using React, if we need to test that we would have to test all its functionality

Reply:
While I would normally agree that testing passing props is pointless, I'd argue that testing that the button renders correctly with it's children is actually testing the functionality of the button. As a result it would help prevent changes that could later be made to the button, like if they delete or remove the children inside.